### ISSUE TITLE
system uuid fail over bug during install

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#1424]
Thanks to @fulanodoe in the forum for looking into this one. On some motherboards the UUID is not set and the intended fail over mechanism seems not to be coping with this, resulting in the following error during install:
```
Exception while running command(['/', 'u', 's', 'r', '/', 'b', 'i', 'n', '/', 'h', 'o', 's', 't', 'i', 'd']): [Errno 13] Permission denied
```
Quoting @fulanodoe post:
https://forum.rockstor.com/t/i-can-not-gets-past-the-initial-set-up/1887/4

"Found the problem/solution. For whatever reason the portion of /opt/rockstor/src/rockstor/system/osi.py at the end that tried to get a system UUID dies if there is not one set at "/sys/class/dmi/id/product_uuid", instead of getting it from hostid as intended, this seems to break the rest of the setup process."

Their workaround was to set the motherboards UUID.

**Comment by Termitebug** [comment]
Hello, Is there yet any progress on this issue? I can't reinstall Rockstor on my Acer Aspire M3201 because of this issue.

**Comment by phillxnet** [comment]
Linking to possibly related recent commit:
https://github.com/rockstor/rockstor-core/pull/1461/commits/101c633769f32e5cb7a9f0aeec6f1d5950b42344

**Comment by MFlyer** [comment]
To @schakrava & @phillxnet 
This is not Rockstor related at all, but a bios issue: if not uuid users had to set it/patch it via bios or manually set on a VM

Mirko

**Comment by phillxnet** [comment]
@MFlyer I agree in part and maybe we should add this as a minimum spec to the docs which would give users an indication of it's use, ie to ease re-installation / re-registration on the same machine but it's still the case that there is a fail over mechanism which isn't working as intended, at least that's my understanding. So I see this issue as addressing this failing fail over mechanism.
